tempo/k8s/utils.py

```diff
@@ -115,3 +118,90 @@ def get_logs_insights_message_dumper():
     api_instance = client.CoreV1Api()
     response = api_instance.read_namespaced_pod_log(DefaultInsightsServiceName, DefaultSeldonSystemNamespace)
     return response
+
+
+def deploy_redis():
```

Comment:
Is the `deploy_redis` method more of a utility for quick testing or would we expect users leveraging this method to deploy Redis?